web/modules/custom/nys_metatags/nys_metatags.module

```diff
@@ -0,0 +1,165 @@
+  if (!($node instanceof NodeInterface)) {
+    return;
+  }
+  if ($node->bundle() === 'honoree') {
+    $title = $node->title->value ?? '';
+    if (!$title) {
+      return;
+    }
+    $type = $node->field_honoree_type[0]->value;
+    foreach ($metatag_attachments['#attached']['html_head'] as $id => $attachment) {
+      if ($attachment[1] == 'title') {
+        if ($type === 'women') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $title . ' | Women of Distinction | NYSenate.gov';
+        }
+        elseif ($type === 'veteran') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $title . ' | Veteran\'s Hall of Fame | NYSenate.gov';
+        }
+      }
+      if ($attachment[1] == 'og_title' || $attachment[1] == 'twitter_cards_title') {
+        if ($type === 'women') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $title . ' | Women of Distinction';
+        }
+        elseif ($type === 'veteran') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = $title . ' | Veteran\'s Hall of Fame';
+        }
+      }
+    }
+  }
+  if ($node->bundle() === 'microsite_page') {
+    $title = $node->title->value ?? '';
+    if (!$title) {
+      return;
+    }
+    $type = $node->field_microsite_page_type[0]->target_id;
+    $senator_id = $node->field_senator_multiref[0]->target_id;
+    try {
+      $senator = \Drupal::entityTypeManager()->getStorage('taxonomy_term')
+        ->load($senator_id);
+    }
+    catch (\Throwable) {
+      $senator = NULL;
+    }
+
+    if (isset($senator->name->value) && !empty($senator->name->value)) {
+      $senator_name = $senator->name->value;
+    }
+
+    foreach ($metatag_attachments['#attached']['html_head'] as $id => $attachment) {
+      if ($attachment[1] == 'title') {
+        if ($type == '200002' || $type == '200003' || $type == '200004') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name . '\'s ' . $title . ' | NYSenate.gov';
+        }
+        elseif ($type == '200005') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name . ' | NYSenate.gov';
+        }
+      }
+
+      if ($attachment[1] == 'og_title' || $attachment[1] == 'twitter_cards_title') {
+        if ($type == '200002' || $type == '200003' || $type == '200004') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name . '\'s ' . $title;
+        }
+        elseif ($type == '200005') {
+          $metatag_attachments['#attached']['html_head'][$id][0]['#attributes']['content'] = 'Senator ' . $senator_name;
+        }
+      }
+    }
+
+    if (!empty($node->field_thumbnail[0]->target_id)) {
+      $mid = $node->field_thumbnail[0]->target_id;
+      try {
```

Comment:
you can avoid using storage and `try..catch` here with:

`$media = $node->field_thumbnail[0]->entity ?? NULL;`